wallet/src/main/scala/org/bitcoins/wallet/WalletCallbacks.scala

```diff
@@ -15,6 +16,10 @@ import scala.concurrent.{ExecutionContext, Future}
   */
 trait WalletCallbacks {
 
+  def onBlockTransactionProcessed: CallbackHandler[
```

Comment:
Can you add a scaladoc on when this is called? This is done after all "internal" wallet processing is done right?